### DISCUSSION TITLE
Sorted with Dataview

### DISCUSSION THREAD

**Discussion by menagerie198** [alexandru-dinu/obsidian-sortable#10] [Ideas]
Hi, hope this is the right place for this, delete otherwise: Should Sorted work with tables created via Dataview? They are not actually tables in the markdown file, so maybe not. If this is not a bug, could it be made into a feature?

**Comment by alexandru-dinu** [comment]
Hello,

Sortable works with dataview tables because it just sorts the HTML row elements (`tr`) of a table's body, disregarding the class that dataview adds to its tables:
```html
<table class="dataview table-view-table">
```
An immediate fix would be to add a setting for sortable to ignore dataview tables.

Also, there's an open discussion about this topic [here](https://github.com/blacksmithgu/obsidian-dataview/discussions/136).
Any idea or suggestion is greatly appreciated!

**Reply by menagerie198** [reply]
In that case, I have a bug. When I sort a dataview table (128 rows, 4 cols), the table sorts as desired, but after a short pause (1 sec or so) it reverts to its unsorted order. I have tried this in my vault, with many plugins, snippets and the Minimal theme, and also in a vanilla test vault - same result. Do you need any more info from me?

**Reply by alexandru-dinu** [reply]
This is a known compatibility [issue](https://github.com/alexandru-dinu/obsidian-sortable/issues/6#issuecomment-825475321). From dataview [0.2.14](https://github.com/blacksmithgu/obsidian-dataview/releases/tag/0.2.14) onwards an auto-refresh feature was added, which disregards any past state of the table.
